kolibri/plugins/management/assets/src/views/data-page/index.vue

```diff
@@ -14,7 +14,7 @@
         {{$tr('detailsSubHeading')}}
       </p>
       <form :action="sessionlogurl" method="get">
```

Comment:
Maybe change these to `@submit`, this is the only place in the code where `action` attribute is used.

Reply:
@jonboiser Addressed in the latest commit.